### DISCUSSION TITLE
Tutorial Ideas

### DISCUSSION THREAD

**Discussion by kecnry** [phoebe-project/phoebe2#759] [Ideas]
Share ideas for tutorials (or if you're looking to contribute to phoebe, here are some ideas for tutorials to write and contribute to the documentation).

**Comment by kecnry** [comment]
Create a line profile at a specific time, and create a custom cost function during fitting which measures the FWHM of the synthetic line profile and includes as an additional prior based on a value measured on real observations.

**Comment by kecnry** [comment]
example showing extraction of absolute temperatures using `pblum_mode='dataset-coupled'`

**Comment by kecnry** [comment]
create a custom feature that modifies the light curve to add a sinusoidal component before the cost-function is computed (for a non-constant third light contribution, for example).

**Comment by kecnry** [comment]
annotate animations via animation callbacks

**Comment by westobin** [comment]
Phase aliasing to show data across phases from -0.6 to 0.6 rather than being limited to only -0.5 to 0.5.

**Reply by kecnry** [reply]
Unfortunately this is not natively supported in phoebe (although could be done manually in matplotlib, etc) and is surprisingly difficult to implement with alongside all the other plotting functionality.  See https://github.com/phoebe-project/phoebe2/issues/9

**Comment by kecnry** [comment]
in-depth tutorial for managing complex constraints (pot, requiv, fillout-factor, q for contacts, kepler's third law).  See https://github.com/phoebe-project/phoebe2/discussions/824 for an example.

**Comment by kecnry** [comment]
Ability to extract per-component light curves (see https://github.com/phoebe-project/phoebe2/discussions/839#discussioncomment-8954138)

**Comment by mas0nr** [comment]
Basic usage of the dynesty backend in place of MCMC
